.tavis.yml

```diff
@@ -0,0 +1,21 @@
+dist: xenial
+sudo: false
+
+language: generic
+
+env:
+  matrix:
+    - K8S_VERSION=v1.15.0 KONG_TEST_DATABASE=postgres
+    - K8S_VERSION=v1.15.0 KONG_TEST_DATABASE=cassandra
+    - K8S_VERSION=v1.14.3 KONG_TEST_DATABASE=postgres
+    - K8S_VERSION=v1.13.6 KONG_TEST_DATABASE=postgres
+matrix:
+  allow_failures:
+    - env: K8S_VERSION=v1.13.6
```

Comment:
Why is this allowed?

Reply:
I found when implementing 1.13.6 tests via k8s manifests that they're not passing reliably. It's on my plate to fix it but having this test would have caught it earlier